### ISSUE TITLE
Do you have a code sample that fail with jest-axe but won't be catch by jsx-a11y eslint plugin?

### ISSUE THREAD

**Issue by edouard-lopez** [nickcolley/jest-axe#157]
Thanks for the lib, I love it!

I would like to demonstrate the package to my teammates, but I need a convincing example. The one on the doc using `<img>` tag with an `alt` attribute is already catched by `eslint-plugin-jsx-a11y` in the IDE.

Could you provide a React/HTML snippet that would trigger `jest-axe` but not `jsx-a11y`?

**Comment by damianpoole** [comment]
On quick inspection, it looks like `jsx-a11y` uses `axe` under the hood so I would expect both this and `jsx-ally` to fail tests for the same issues.

**Comment by nickcolley** [comment]
Yeah think if all your teammates are using the same plugin then maybe you wont get much from this package, however there's a good chance they dont so thats where having it checked by your continuous integration build is useful.
